### PR TITLE
Improve configuration files and README.md

### PR DESCRIPTION
**Pull request by bzumik1**
Make special configuration for Plug&Play version so it is even easier for new joiners.

### CODE REVIEW

File: config-plug-and-play.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+sensor:
+  # Shows WiFi signal strength in Home Assistant
+  - platform: wifi_signal
+    name: "${name} signal"
+    update_interval: 10s
+  
+  # Shows run time in Home Assistant
+  - platform: uptime
+    name: "${name} uptime"
```
Comment (lanwin) [inline config-plug-and-play.yaml:125]: I think this should not be are. This are custom stuff not everyone needs.

File: config-plug-and-play.yaml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        name: "AC Unit 2 - room temperature"
+      # Only supported on NASA devices for now
+      room_humidity:
+        name: "AC Unit 2 - humidity"
+      target_temperature:
+        name: "AC Unit 2 - target temperature"
+      power:
+        name: "AC Unit 2 - power switch"
+      mode:
+        name: "AC Unit 2 - mode"
```
Comment (lanwin) [inline config-plug-and-play.yaml:104]: To make it more clear we should remove the comments here and add just the basic climate control and the comment to change the address.

File: config-plug-and-play.yaml [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+samsung_ac:
+  devices:
+    # AC UNIT 1 - add first indoor unit
+    # unit address, for non-NASA devices the format will be like "c4", the address can be found in logs on start
+    - address: "20.00.00" # !!!CHANGE ME!!!
+    
+    # Each property below is optional (climate, room_temperature etc.) - you can delete those which are not needed
+      # Creates climate in Home Assistant
+      climate:
+        name: "AC Unit 1"
```
Comment (lanwin) [inline config-plug-and-play.yaml:66]: I would suggest to use a room name instead of "AC Unit 1". I would guess most people use room names here.

**Review by lanwin — CHANGES_REQUESTED** [review]
Ok after rethinking this it was a bad idea to split the examples. Could you please join them back. 

Also the logger stuff should be copied over from the DIY example. Since this is a mayor problem when you use a NodeMCU or a D1 Mini.

**Comment by bzumik1** [comment]
Will do

**Comment by lanwin** [comment]
Thak you @bzumik1 I have decided to discard the DIY stuff and updated the readme based on your seggstions. So this can be closed.

**Pull request closed without merging**